WebHostLib/api/tracker.py

```diff
@@ -0,0 +1,166 @@
+from datetime import datetime, timezone
+from typing import Any
+from uuid import UUID
+
+from flask import abort
+
+from NetUtils import ClientStatus, NetworkItem, SlotType
+from WebHostLib import cache
+from WebHostLib.api import api_endpoints
+from WebHostLib.models import Room
+from WebHostLib.tracker import TrackerData
+
+
+@api_endpoints.route("/tracker/<suuid:tracker>")
+@cache.cached(timeout=60)
+def tracker_data(tracker: UUID) -> dict[str, Any]:
+    """
+    Outputs json data to <root_path>/api/tracker/<id of current session tracker>.
+
+    :param tracker: UUID of current session tracker.
+
+    :return: Tracking data for all players in the room. Typing and docstrings describe the format of each value.
+    """
+    room: Room | None = Room.get(tracker=tracker)
+    if not room:
+        abort(404)
+
+    tracker_data = TrackerData(room)
+
+    all_players: dict[int, list[int]] = tracker_data.get_all_players()
+
+    groups: list[list[dict[str, int | list[dict[str, str | int | list[int]]]]]] = []
```

Comment:
dict[str, str | int | list[int]] is a rather awkward type, this might be a good time to use typing.TypedDict instead, so typing can be applied on the individual values based on keys.